src/client/api/restful/get_outputs_id.c

```diff
@@ -407,76 +551,153 @@ static char *compose_api_command(outputs_query_list_t *list, char const *const a
   return cmd_buffer;
 }
 
-int get_outputs_id(iota_client_conf_t const *conf, outputs_query_list_t *list, res_outputs_id_t *res) {
-  if (conf == NULL || res == NULL) {
+int get_basic_outputs(iota_client_conf_t const *conf, char const *const indexer_path, outputs_query_list_t *list,
+                      res_outputs_id_t *res) {
+  if (conf == NULL || indexer_path == NULL || res == NULL) {
     // invalid parameters
     return -1;
   }
 
+  if (strcmp(indexer_path, INDEXER_API_PATH)) {
```

Comment:
`strcmp` returns negative numbers also. (this comparison occurs in multiple places in commited code)
```suggestion
  if (strcmp(indexer_path, INDEXER_API_PATH) != 0) {
```

Reply:
Thanks Matjaz.